crates/ark/tests/data_explorer.rs

```diff
@@ -1856,7 +1856,10 @@ fn test_frequency_table() {
             values: format_string(
```

Comment:
You could use `format_column` here instead, which would also report the Special codes if necessary.